packages/typoas-generator/src/generator/utils/__tests__/__snapshots__/enums.spec.ts.snap

```diff
@@ -0,0 +1,15 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`canConvertSchemaToEnum should handle string schema 1`] = `true`;
+
+exports[`canConvertSchemaToEnum should not handle non string schemas 1`] = `false`;
+
+exports[`canConvertSchemaToEnum should not handle string schema without enum 1`] = `false`;
```

Comment:
No need to create snapshot for those, snapshots are mainly for the generated stuff